src/cli/cli.ts

```diff
@@ -341,7 +343,9 @@ async function launchContext(options: Options, headless: boolean, executablePath
 
   if (options.proxyServer) {
     launchOptions.proxy = {
-      server: options.proxyServer
+      server: options.proxyServer,
+      username: options.proxyUsername,
+      password: options.proxyPassword
```

Comment:
```suggestion
     ...(options.proxyUsername && {username: options.proxyUsername}),
     ...(options.proxyPassword && {password: options.proxyPassword}),
```
Something like this should make the bots happy. They are trying to assert the generated code to the code in the tests and have probably currently undefined or null values from it.

Reply:
Thank you. As you pointed out, the added parameter was giving an error in the test.